casepro/settings_common.py

```diff
@@ -57,6 +57,7 @@
 SITE_EXTERNAL_CONTACT_URL = 'http://localhost:8001/contact/read/%s/'
 SITE_BACKEND = 'casepro.backend.NoopBackend'
 SITE_ANON_CONTACTS = False
+SITE_CASE_RESPONSE_REQUIRED_TIME = 1440  # 24 hours in minutes
```

Comment:
I'm wondering if we should set this to `None` as the default setting, so that it doesn't affect existing casepro installations.